python/pyspark/sql/session.py

```diff
@@ -557,8 +557,9 @@ def _create_from_pandas_with_arrow(self, pdf, schema, timezone):
 
         # Create Arrow record batches
         safecheck = self._wrapped._conf.arrowSafeTypeConversion()
+        col_by_name = True  # col by name only applies to StructType columns, can't happen here
```

Comment:
Meaning the value of `col_by_name` doesn't matter here, right?

Reply:
Yes, that's right. This will be removed when we take out that conf.